pabi_procurement/models/purchase_web_interface.py

```diff
@@ -147,20 +147,19 @@ def send_pbweb_action_request_test(self, request_name, action, user_name):
 
     @api.model
     def send_pbweb_action_request(self, request, action):
-        users = self.env['res.users'].search([('id', '=', self._uid)])
-        assert len(request) == 1, \
-            "Only 1 Purchase Request could be done at a time."
         ConfParam = self.env['ir.config_parameter']
+        if ConfParam.get_param('pabiweb_active') != 'TRUE':
```

Comment:
I add this pabiweb_active = FALSE to allow bypassing Alfresco (for testing purposes)